README.md

```diff
@@ -1,2 +1,106 @@
 # python-aodnfetcher
 Multi-protocol artifact fetcher library/utility
+
+## Usage
```

Comment:
Should probably have an 'installation' heading as well

Reply:
No problem. It's entirely standard in terms of packaging/installing, but it never hurts to add a "quick start" type reminder.